selfdrive/car/body/values.py

```diff
@@ -16,23 +17,35 @@ class CarControllerParams:
 
 class CAR:
   BODY = "COMMA BODY"
+  BODY_KNEE = "COMMA BODY WITH KNEE"
 
 CAR_INFO: Dict[str, CarInfo] = {
   CAR.BODY: CarInfo("comma body", package="All", harness=Harness.none),
+  CAR.BODY_KNEE: CarInfo("comma body + knee", package="All", harness=Harness.none),
```

Comment:
this is just for docs/harness, don't think we want another entry there
```suggestion
  CAR.BODY_KNEE: None,
```